### PR TITLE
Fix: #1320 Release Notes for 1.11

### LINKED ISSUE #1320: Release Notes for 1.11
- Added quick action for New Private Tab from home screen. ([#1258](https://github.com/brave/brave-ios/issues/1258))  
- Improved video orientation on Youtube. ([#1189](https://github.com/brave/brave-ios/issues/1189))
- Fixed passcode not being reset when resuming from the background. ([#1203](https://github.com/brave/brave-ios/issues/1203))
- Fixed browser becoming unresponsive when using print preview in certain conditions. ([#1144](https://github.com/brave/brave-ios/issues/1144))

### PR DESCRIPTION
**Pull request by srirambv**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [ ] *Unit Tests* are updated to cover new or changed functionality
- [ ] User-facing strings use `NSLocalizableString()`
- [ ] New files have MPL-2.0 license header.

## Test Plan:

Fixes https://github.com/brave/brave-ios/issues/1320

## Reviewer Checklist:

- [ ] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [ ] Issues are assigned to at least one epic.
- [ ] Issues include necessary QA labels:
  - [ ] `QA/(Yes|No)`
  - [ ] `release-notes/(include|exclude)`
  - [ ] `bug` / `enhancement`
- [ ] Necessary security reviews have taken place.
- [ ] Adequate test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.11](https://github.com/brave/brave-ios/releases/tag/v1.11)
+ 
+ - Added quick action for New Private Tab from home screen. ([#1258](https://github.com/brave/brave-ios/issues/1258))  
+ - Improved video orientation on Youtube. ([#1189](https://github.com/brave/brave-ios/issues/1189))
+ - Fixed passcode not being reset when resuming from the background. ([#1203](https://github.com/brave/brave-ios/issues/1203))
+ - Fixed browser becoming unresponsive when using print preview in certain conditions. ([#1144](https://github.com/brave/brave-ios/issues/1144))
+
+
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: extra space

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.11](https://github.com/brave/brave-ios/releases/tag/v1.11)
+ 
+ - Added quick action for New Private Tab from home screen. ([#1258](https://github.com/brave/brave-ios/issues/1258))  
+ - Improved video orientation on Youtube. ([#1189](https://github.com/brave/brave-ios/issues/1189))
+ - Fixed passcode not being reset when resuming from the background. ([#1203](https://github.com/brave/brave-ios/issues/1203))
```
Comment (kjozwiak) [inline CHANGELOG.md]: nit: I think `when resuming from background` sounds better compared to `when resuming from the background`

**Review by kjozwiak — CHANGES_REQUESTED** [review]
Just some small nits

File: CHANGELOG.md (hunk 2 above)
Comment (srirambv) [reply CHANGELOG.md]: changed

File: CHANGELOG.md (hunk 1 above)
Comment (srirambv) [reply CHANGELOG.md]: removed

**Review by kjozwiak — APPROVED** [review]
LGTM 👍

**Pull request merged**